boa_engine/src/vm/opcode/generator/mod.rs

```diff
@@ -225,3 +217,99 @@ impl Operation for GeneratorNextDelegate {
         }
     }
 }
+
+/// `AsyncGeneratorNextDelegate` implements the Opcode Operation for `Opcode::AsyncGeneratorNextDelegate`
+///
+/// Operation:
+///  - Delegates the current async generator function to another async generator.
+#[derive(Debug, Clone, Copy)]
+pub(crate) struct AsyncGeneratorNextDelegate;
+
+impl Operation for AsyncGeneratorNextDelegate {
+    const NAME: &'static str = "AsyncGeneratorNextDelegate";
+    const INSTRUCTION: &'static str = "INST - AsyncGeneratorNextDelegate";
+
```

Comment:
Would it be possible to document (maybe put a link) from where in the spec is this taken?

Reply:
I remove it because it'll be better to generalize `GeneratorNextDelegate` as in `iterator_close` instead.